contracts/KetherNFT.sol

```diff
@@ -0,0 +1,115 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8;
+
+// FIXME: Use 4.x pre-release which slims down the 721 implementation
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+
+import "./IKetherHomepage.sol";
+
+
+// TODO: Name this something really cool
+contract Wrapper {
+  constructor(address target, bytes memory payload) {
+    (bool success,) = target.call(payload);
+    require(success, "Wrapper: target call failed");
+
+    selfdestruct(payable(target));
```

Comment:
Is this who we want to be transferring to? See my idea for the bailout, I think this should be the owner not the target

Reply:
I don't think it particularly matters, since this address shouldn't have a balance anyway but it's a required field for selfdestruct. This creates an easy way to give KetherHomepage a tip for using the NFT dapp. :P